vagrant/Install-on-Ubuntu-22.sh

```diff
@@ -197,7 +197,7 @@ if [ "x$2" == "xinstall-nginx" ]; then #DOCS:
 sudo tee /etc/php/8.1/fpm/pool.d/www.conf << EOF_PHP_FPM_CONF
 [www]
 ; Replace the tcp listener and add the unix socket
-listen = /var/run/php8.1-fpm.sock
+listen = /var/run/php-fpm.sock
```

Comment:
This change is missing in the two other scripts.

Reply:
Good idea, I changed that now but without the `-www` part. You can doublecheck with `grep -r '\.sock' docs/ vagrant/`